### DISCUSSION TITLE
Mac and OBS encoder question

### DISCUSSION THREAD

**Discussion by Maclife420** [obsproject/obs-studio#5090] [Development Questions]
Hey there,

I am having a question. On a Mac you can choose between x264, Apple-VT-h264 Softwareencoder, andApple-VT-h264 Hardwareencoder so what is the real difference? I guess
- x264 just uses your Intel CPU power
But what about those other 2 options is the Apple-VT-h264 Hardwareencoder using your AMD graphics card ? Or your Intel Iris cpu integrated graphics card ???

**Comment by Fenrirthviti** [comment]
Please use the forums or Discord for support. Thanks!
